templates/csi-daemonset.yaml

```diff
@@ -55,7 +55,11 @@ spec:
           imagePullPolicy: {{ .Values.csi.image.pullPolicy }}
           args:
             - --endpoint=/provider/vault.sock
-            - --debug={{ .Values.csi.debug }}
+            {{- if .Values.csi.debug }}
+            - --log-level=debug
+            {{- else }}
+            - --log-level={{ .Values.csi.logLevel }}
+            {{- end }}
```

Comment:
I wonder if we should flip the precedence here?

Reply:
@tvoran  I believe we want to honor `.csi.debug` if it is set, no? If not, then we would be changing the user's current debug setup.

Reply:
@benashz Yeah I wasn't suggesting removing `.csi.debug`, just to prefer `.csi.logLevel` if both `.csi.debug` and `.csi.logLevel` happen to be set. But I was forgetting that's hard to do with helm since logLevel has a default non-empty value, so 👍